src/Mbstring/Mbstring.php

```diff
@@ -138,6 +138,128 @@ public static function mb_encode_mimeheader($s, $charset = null, $transferEncodi
         trigger_error('mb_encode_mimeheader() is bugged. Please use iconv_mime_encode() instead', E_USER_WARNING);
     }
 
+    public static function mb_decode_numericentity($s, $convmap, $encoding = null)
+    {
+        if (null !== $s && !is_scalar($s) && !(is_object($s) && method_exists($s, '__toString'))) {
+            trigger_error('mb_decode_numericentity() expects parameter 1 to be string, '.gettype($s).' given', E_USER_WARNING);
+            return null;
+        }
+
+        if (!is_array($convmap) || !$convmap) {
+            return false;
+        }
+
+        if (null !== $encoding && !is_scalar($encoding)) {
+            trigger_error('mb_decode_numericentity() expects parameter 3 to be string, '.gettype($s).' given', E_USER_WARNING);
+            return '';  // Instead of null (cf. mb_encode_numericentity).
+        }
+
+        if ('' === $s .= '') {
```

Comment:
please don't cast to string like that

Reply:
@keradus: I understand that some might not find it readable; however, I have tried to imitate the style of the project (see `mb_convert_case`). Thank you for bringing this up!